### PR TITLE
Start at using dashboard icons in app builder page

### PR DESCRIPTION
**Pull request by kaapstorm**
The styling needs to be turned into less and moved out of the header.

@biyeun, cc @snopoke 

For context, see [FB 168663](http://manage.dimagi.com/default.asp?168663)

Screenshot:
app_builder_page

### CODE REVIEW

**Comment by czue** [comment]
yikes, this was quite a bit more complicated than I expected! i'll let @biyeun review

**Comment by biyeun** [comment]
shouldn't this use the centered column layout, not the two column layout?

File: corehq/apps/app_manager/templates/app_manager/apps_base.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {% extends 'hqwebapp/two_column.html' %}
```
Comment (biyeun) [inline corehq/apps/app_manager/templates/app_manager/apps_base.html]: what's your plan to move this to the bootstrap 3 version?

**Comment by biyeun** [comment]
I'm assuming this is in an intermediate stage due to all the in-page css and not inheriting from the parent template. what's the plan for implementing this in the more ideal way?

**Comment by kaapstorm** [comment]
I think the left column is the sidebar ... and I think we need it ... but I'd be happy to be wrong, because if we could use Bootstrap 3 it would save a lot of hassle. 

It's definitely in an intermediate stage. The CSS needs to become LESS and it needs to move out.

I don't know a plan. I was kinda hoping you had a plan. Is there a plan? :grin: If not, we should come up with one.

**Comment by kaapstorm** [comment]
Considering the sidebar is empty, the only thing this page does that the dashboard doesn't have is a modal box for undoing deleted apps. Could we add that to the dashboard, and drop this page?

**Comment by kaapstorm** [comment]
Going with [PR 7045](https://github.com/dimagi/commcare-hq/pull/7045) instead.

**Pull request closed without merging**